docs/pages/management/security/proxy-protocol.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: PROXY protocol
+description: How to securely configure PROXY protocol usage with Teleport
+---
+<Admonition type="note" scope={["cloud", "team"]}>
+Users of Teleport Cloud don't need to manage PROXY protocol setting manually. It's set to `on` automatically 
+and can't be changed, since cloud Proxy/Auth services always run behind L4 load balancer with enabled PROXY protocol. 
+</Admonition>
+
+## What is PROXY protocol
+
+The PROXY Protocol is a prefix added to the TCP connection containing information about the client IP. 
+It is most commonly used when networking setup includes a Layer 4 (L4) load balancer between the user and the endpoint service,
+like Teleport Proxy/Auth.
+L4 load balancers, by design, do not retain the original client's IP address and port when forwarding the connection and 
+PROXY Protocol allows to overcome this problem by adding the client's original IP address and port before the actual TCP stream.
```

Comment:
We should mention that Teleport uses the PROXY Line for:
- IP pinning 
- Audit logging 

If you don't set it correctly, the integrity of audit log can be affected